apps/term-apply/pkg/applicant/resume.go

```diff
@@ -2,27 +2,24 @@ package applicant
 
 import (
 	"fmt"
-	"io/ioutil"
 	"log"
-	"os"
-	"strings"
 	"sync"
+
+	"github.com/nebulaworks/orion/apps/term-apply/pkg/s3file"
 )
 
+const RESUME_ROOT_KEY = "/term-apply/resumes/"
```

Comment:
I think we try to consolidate some of these constants at some point but I see why its easier for the time being

Reply:
100% agree, I opened inf 515 when it go too unwieldy to do as part of this PR.